src/pages/FolderPage.tsx

```diff
@@ -0,0 +1,53 @@
+import FolderList from '../components/FolderPage/FolderList';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import AddLinkBar from '../components/FolderPage/AddLinkBar';
+import SearchBar from '../components/SearchBar';
+import { useEffect, useRef, useState } from 'react';
+
+const FolderPage = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isAddLinkBarVisible, setIsAddLinkBarVisible] = useState(true);
+
+  const addLinkBarRef = useRef(null);

```

Comment:
```suggestion
const addLinkBarRef = useRef<HTMLDivElement>(null);
```

`useRef`가 DOM을 조작할때 쓰이는 경우가 많은데 `addLinkBarRef.current`로 얻는 HTML 요소들을 조작할때 타입이 있으면 편한 경우가 종종 있어서 어떤 타입의 ref인지 명시해주는 타입을 적어주세요.